src/sentry/toolbar/views/iframe_view.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
+
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+from sentry.toolbar.utils.url import check_origin
+from sentry.web.frontend.base import OrganizationView, region_silo_view
+
+logger = logging.getLogger(__name__)
+
+
+@region_silo_view
+class IframeView(OrganizationView):
+    security_headers = {
+        "X-Frame-Options": "ALLOWALL"
+    }  # allows response to be embedded in an iframe.
+
+    def respond(self, template: str, context: dict[str, Any] | None = None, status: int = 200):
+        response = super().respond(template, context=context, status=status)
+        for header, val in IframeView.security_headers.items():
+            response[header] = val
```

Comment:
Let's just assign these statically.  I.e. without the class variable.  `response["X-Frame-Options"] = "ALLOWALL"`